python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -70,6 +70,7 @@ class ImageDifferenceConfig(pexConfig.Config):
     doMatchSources = pexConfig.Field(dtype=bool, default=True,
         doc="Match diaSources with input calexp sources and ref catalog sources")
     doMeasurement = pexConfig.Field(dtype=bool, default=True, doc="Measure diaSources?")
+    doMeasurementNew = pexConfig.Field(dtype=bool, default=True, doc="Measure diaSources using new algorithm?")
```

Comment:
OK, done. Also changed docstring to: "Fit dipoles using new algorithm?"
Note this option will be removed in the future once DipoleMeasurement is slated for removal.